Listenbrainz.xcodeproj/project.pbxproj

```diff
@@ -283,14 +289,14 @@
 		6F7620A329C66C8000CDA3E8 /* Screens */ = {
 			isa = PBXGroup;
 			children = (
+				BBFC06722C1EE2E400668E90 /* Search Users */,
```

Comment:
File names should not have gaps. Keep it SearchUsers only

Reply:
@akshaaatt , it is the group name, changed it.